### PR TITLE
Pl 1791

### PR DESCRIPTION
**Pull request by grace09thompson**
I switched the transfer request to be GBP and faster payments simply because a lot of the devs that are looking at this are focused on UK, but if you think we should keep it as Sepa, just let me know and I'll revert back.

### CODE REVIEW

File: client.js [hunk 1]
```diff
@@ -1,41 +1,141 @@
...
+    // get button placeholder element
+    var element = document.getElementById(elementId);
+
+    // create the button
+    button = window.Token.createTokenButton(element, {
+        label: "Redirect Token Quick Checkout",
+    });
+
+    // bind the Token Button to the Redirect Controller when ready
+    tokenController.bindButtonClick(button, function(action) {
```
Comment (colin-token) [inline client.js]: do we need onload here too?

File: client.js [hunk 2]
```diff
@@ -1,41 +1,141 @@
...
+    });
+}
+
+function getTokenRequestUrl(done) {
     var XHR = new XMLHttpRequest();
 
-    // Set up our request
-    XHR.open('POST', 'http://localhost:3000/transfer', true);
+    //set up the transfer request
+    XHR.open("POST", "http://localhost:3000/transfer", true);
```
Comment (colin-token) [inline client.js]: single quotes

File: server.js [hunk 3]
```diff
@@ -45,19 +45,16 @@ function initServer(member, alias) {
...
-                    .then(function (token) {
-                        // Redeem the token to move the funds
-                        member.redeemToken(token, 4.99, 'EUR', 'Book Purchase')
-                            .then(function (transfer) {
-                                console.log('\n Reedeem Token Response:', transfer);
-                                res.status(200);
-                                res.json(transfer)
-                            });
-                    })
+        var tokenId = req.query.tokenId;
```
Comment (colin-token) [inline server.js]: why did you take out parse?

Comment (grace09thompson) [reply server.js]: because it doesn't work with the popup - unless this has been fixed?

File: client.js (hunk 1 above)
Comment (grace09thompson) [reply client.js]: I'm not entirely sure why, but when I add onLoad to this one, it doesn't work...

File: server.js (hunk 3 above)
Comment (grace09thompson) [reply server.js]: https://tokenio.atlassian.net/browse/RD-747

**Pull request merged**